packages/lesswrong/components/users/NewUserCompleteProfile.tsx

```diff
@@ -131,10 +139,7 @@ const NewUserCompleteProfile: React.FC<NewUserCompleteProfileProps> = ({ current
   return <SingleColumnSection>
     <div className={classes.root}>
       <Typography variant="display2" gutterBottom className={classes.title}>
-        Thanks for registering for {siteNameWithArticleSetting.get()}
-      </Typography>
-      <Typography variant='body2'>
-        Take a moment to complete your profile
+        Thanks for registering! Complete your profile.
```

Comment:
I didn't forum-gate these changes since I'm pretty sure LW doesn't use this page, but let me know if I'm mistaken (@b0b3rt FYI)

Reply:
Yeah, I'm pretty sure this is an EAForum-only page which depends on `usernameUnset` being set to true for the user (which only seems to happen for auth0 registrations).